packages/mux-video/src/index.ts

```diff
@@ -248,6 +312,7 @@ class MuxVideoElement extends CustomVideoElement<HTMLVideoElementWithMux> {
 
       mux.monitor(this.nativeEl, {
         debug,
+        beaconDomain,
```

Comment:
Confirmed `beacon-domain` works as expected when supplied and when it's not supplied then `mux-embed` uses the default